sabre/dav/lib/CardDAV/Backend/PDO.php

```diff
@@ -198,7 +198,7 @@ function deleteAddressBook($addressBookId) {
         $stmt = $this->pdo->prepare('DELETE FROM ' . $this->addressBooksTableName . ' WHERE id = ?');
         $stmt->execute([$addressBookId]);
 
-        $stmt = $this->pdo->prepare('DELETE FROM ' . $this->addressBookChangesTableName . ' WHERE id = ?');
+        $stmt = $this->pdo->prepare('DELETE FROM ' . $this->addressBookChangesTableName . ' WHERE addressbookid = ?');
```

Comment:
@DeepDiver1975 the update comes with this "bonus" fix, would that help with any other known issues ?